docs/client.rst

```diff
@@ -72,18 +72,111 @@ WBEM Operation recording
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
Further up in the `client.rst` file, there is the text: 

WBEM Operation recording - operation_recorder is an attribute of WBEMConnection that allows recording of WBEM operations

That is no longer correct. Please adjust.

Reply:
Agreed. Changed to a general statement

Class :class:`~pywbem.WBEMConnection`
    includes the capability to optionally record the WBEMConnection method
    calls that communicate with a WBEM server including creating yaml
    files for test generation and logging using the Python logging facility.